server/util/getSessionCookies.js

```diff
@@ -1,10 +1,20 @@
 const fetch = require('./fetch');
 const setCookieParser = require('set-cookie-parser');
 
+const getCookieString = cookies => {
+	return Object.keys(cookies)
+		.map(key => `${key}=${cookies[key]}`)
+		.join(';');
+};
+
 module.exports = function getSessionCookies(url = '', requestCookies = {}) {
 	return new Promise((resolve, reject) => {
 		if (url.length && (!requestCookies.kv || !requestCookies.kvis || !requestCookies.kvbskt)) {
-			fetch(url).then(res => {
+			fetch(url, {
+				headers: {
+					Cookie: getCookieString(requestCookies),
+				},
+			}).then(res => {
```

Comment:
What else might this block be helping with?

Reply:
I suspect this will help keep the sessions in sync and that may help some of the other issues I had seen this week. Will report back after this is out in dev.

Reply:
Any situation where we want to look at cookies that last longer than the session during the session startup process. It seems like something related to the experiment cookies or something similar might be helped by this.